### ISSUE TITLE
please answer

### ISSUE THREAD

**Issue by monnad21** [steam-account-creator/Steam-Account-Generator#55]
the question is whether it is possible to add  database of nicknames and passwords

**Comment by despa1r1337** [comment]
what do you mean?

**Comment by monnad21** [comment]
Well, like you can just add your nickname and password database

**Comment by despa1r1337** [comment]
> can just add your nickname and password database

no, but you can create accounts with your own username and password
![image](https://user-images.githubusercontent.com/48297813/67140734-3158cd00-f266-11e9-94c6-37b7b7cce280.png)

**Comment by despa1r1337** [comment]
solved?

**Comment by EarsKilla** [comment]
🤔 it can be made like user-agents

btw you can little edit code (ofc, if you can) and replace random words with nicknames
